spec/spec_helper.rb

```diff
@@ -229,7 +232,7 @@ def main_app
   end
 
   config.after(:each, type: :view) do
-    WebMock.disable_net_connect!(allow_localhost: true, allow: 'chromedriver.storage.googleapis.com')
+    WebMock.disable_net_connect!(allow_localhost: true, allow: allowed_hosts)
```

Comment:
where does allowed_hosts come from?

Reply:
it's defined further above, in this file. the set had already been applied in initial `WebMock` setup. this reset was causing test order issues whenever view tests are run as part of a larger set.